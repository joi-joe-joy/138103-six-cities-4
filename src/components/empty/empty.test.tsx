import * as React from "react";
import renderer from "react-test-renderer";
import Empty from "./empty";

it(`Render Empty correctly`, () => {
  const tree = renderer.create(
      <Empty/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
