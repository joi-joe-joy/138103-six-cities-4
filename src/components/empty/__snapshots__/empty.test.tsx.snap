// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Empty correctly 1`] = `
<div
  className="cities"
>
  <div
    className="cities__places-container cities__places-container--empty container"
  >
    <section
      className="cities__no-places"
    >
      <div
        className="cities__status-wrapper tabs__content"
      >
        <b
          className="cities__status"
        >
          No places to stay available
        </b>
      </div>
    </section>
    <div
      className="cities__right-section"
    />
  </div>
</div>
`;
